### PR TITLE
cli review connects #673

### PR DESCRIPTION
**Pull request by kosz**

### CODE REVIEW

File: handout/cli/adding-third-party-libraries.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 The CLI provides the ability to integrate third party libraries into the application. For step-by-step information, visit https://github.com/angular/angular-cli/wiki/3rd-party-libs.
 
+> At the time of writing, the angular-cli internals were undergoing a potential change where webpack would be used instead of systemJS for integrating installed third party libraries which would make the setup a lot simpler than currently described in the documentation referenced above.
```
Comment (e-schultz) [inline handout/cli/adding-third-party-libraries.md]: updated docs are here - https://github.com/angular/angular-cli#3rd-party-library-installation

**Comment by gillianml** [comment]
connects to #673

**Comment by kosz** [comment]
yes. @e-schultz, I updated the PR

**Comment by e-schultz** [comment]
connects #673

**Pull request merged**